### ISSUE TITLE
Amazon.Lambda.Tools set values in aws-lambda-tools-defaults.json based on enviroment

### ISSUE THREAD

**Issue by RomanBorysenok** [aws/aws-lambda-dotnet#126]
Hello,

Thank you for your hard work!
I have one question regarding aws-lambda-tools-defaults.json
Is there any way to specify values for deployment based on environment?
Something like aws-lambda-tools-development.json, aws-lambda-tools-staging.json ...
We need such functionality to set "s3-bucket" name based on the env (dev, stage, prod) and publish the serverless Api just by "dotnet lambda deploy-serverless" command, without any arguments.

Thanks!

**Comment by normj** [comment]
I'm working on .NET Core specific documentation on this but hopefully this will unblock you.

The [aws-serverless-samfarm](https://github.com/awslabs/aws-serverless-samfarm) has a sample on how to setup a CI/CD pipeline which defines the different template parameter for each stage in a separate JSON file. This sample is node.js based and in the buildspec.yml file it uses the command

`aws cloudformation package --template $INPUT_FILE --s3-bucket $S3_BUCKET --output-template post-saml.yaml`

Which will package up the application and update the template to where the code was uploaded. This won't work for .NET Core because you need something to actually build the project. I recently released version `1.6.0` of the dotnet CLI tools, `Amazon.Lambda.Tools`, which has a new command `package-ci` which does the equivalent of the command above plus build the project like `deploy-serverless` does. You would replace the command to something like

```
cd ./LambdaCITest && dotnet lambda package-ci --s3-bucket $S3_BUCKET --s3-prefix lambda-ci-build --output-template transformed.template
```

The `--template` parameter is not set because the value is coming from `aws-lambda-tools-defaults.json` like it does for `deploy-serverless`.

**Comment by RomanBorysenok** [comment]
Thank you for your reply! Will try to implement this approach.
